deployments/multiple-ragapps/docker-compose.yml

```diff
@@ -54,19 +54,26 @@ services:
       - KEYCLOAK_ADMIN_PASSWORD=admin
     volumes:
       # Mount example realm data
-      - ./keycloak/example/:/opt/keycloak/data/
+      - ${STATE_DIR:-${PWD}/data}/keycloak/example/:/opt/keycloak/data/
     networks:
       - ragapp-network
+    depends_on:
+      - traefik
 
   manager:
     image: ${MANAGER_IMAGE:-ragapp/manager:latest}
     build: ../../src/manager
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ${STATE_DIR:-${PWD}/data}/manager:/app/config
+      # Mount the ragapps data folder to the manager
+      - ${STATE_DIR:-${PWD}/data}:/app/data
```

Comment:
why do we need two mounts? isn't the folder with the state enough?

Reply:
Removed, thanks